ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11vertex.cpp

```diff
@@ -29,10 +30,6 @@
 #include <map>
 using std::map;
 
-namespace enigma {
```

Comment:
Ahhh... imagine... soon, ENIGMA's globals will each be declared in only one header.

Reply:
That's correct, it will be really awesome when that day finally arrives in total. 😃